config/locales/az/topical_events.yml

```diff
@@ -3,3 +3,16 @@ az:
   topical_events:
     archived:
     featured:
+    lead_organisations:
+      arts_council_england:
```

Comment:
Just a thought: As far as I understand it, these are hacks for special cases (first world war centenary etc) that we want to eventually remove. In this case I wonder if it'd be better to not use translations for them and just hardcode their titles, then we don't have to make any changes to the translation files which we'll later go back and need to remove.

Reply:
It might be a good idea to keep the two hacky logo sections in their own partials (i.e. one for each of the topical events that has these logos).  Everything in those could then be hardcoded, as @BeckaL suggests.

Reply:
Moved into partials as suggested! 👍